src/EnergyPlus/UnitarySystem.cc

```diff
@@ -14247,95 +14312,21 @@ namespace UnitarySystems {
-                                                                0,
-                                                                SpeedRatio);
-                } break;
-                case DataHVACGlobals::Coil_HeatingWater: {
-                    WaterCoils::SimulateWaterCoilComponents(
-                        state, CompName, FirstHVACIteration, this->m_SuppHeatCoilIndex, _, this->m_FanOpMode, PartLoadFrac);
-                } break;
-                case DataHVACGlobals::Coil_HeatingSteam: {
-                    SteamCoils::SimulateSteamCoilComponents(state,
-                                                            CompName,
-                                                            FirstHVACIteration,
-                                                            this->m_SuppHeatCoilIndex,
-                                                            1.0,
-                                                            _,
-                                                            this->m_FanOpMode,
-                                                            PartLoadFrac); // QCoilReq, simulate any load > 0 to get max capacity
-                } break;
-                case DataHVACGlobals::Coil_UserDefined: { // do nothing, user defined coil cannot be controlled
-                    UserDefinedComponents::SimCoilUserDefined(state, CompName, CompIndex, AirLoopNum, HeatingActive, CoolingActive);
-                    if (HeatingActive) PartLoadFrac = 1.0;
-                } break;
-                default:
-                    break;
-                }
-
-                NoLoadTempOut = state.dataLoopNodes->Node(OutletNode).Temp;
-                //      NoOutput = state.dataLoopNodes->Node(InletNode)%MassFlowRate *  &
-                //                       (PsyHFnTdbW(NoLoadTempOut,Node(OutletNode)%HumRat)  &
-                //                        - PsyHFnTdbW(Node(InletNode)%Temp,Node(OutletNode)%HumRat))
-
-                //     If OutletTemp is within ACC of set point, either coil operated or is not needed
-                if (std::abs(state.dataLoopNodes->Node(OutletNode).Temp - DesOutTemp) < Acc ||
-                    this->m_SuppHeatCoilType_Num == DataHVACGlobals::Coil_UserDefined) {
-                    // do nothing, coil is at set point (i.e., gas/elec/steam/user coil will try to hit set point)
-                } else if (PartLoadFrac > 0.0) {
-                    // do nothing, coil tried to hit set point (i.e., gas/elec/steam/user coil tried to hit set point but missed
-                } else if (NoLoadTempOut > (DesOutTemp - Acc)) {
-                    PartLoadFrac = 0.0; // outlet temp > set point, coil is not needed
-                } else {                // outlet temp too low, turn on coil
-
-                    // Get full load result
-                    PartLoadFrac = 1.0;
+                if (this->m_EMSOverrideSuppCoilSpeedNumOn) {
```

Comment:
This part is just adding one if-else, not really changing too much in the else{} branching.

Reply:
OK, it's pretty hard to track on Github's diff view, but I think I'm grokking it.